code-of-conduct/template-code-of-conduct.md

```diff
@@ -0,0 +1,144 @@
+# Code of Conduct template
+
+---
+The Code of Conduct template includes:
+
+* Placeholder text that you can replace with your own text. This text is indicated in {curly braces}.
+* Writing instructions, also in {curly braces}.
+* A supplementary `code-of-conduct-guide.md` that provides more detailed information about using this template. Read the guide document first!

```

Comment:
todo: As @flicstar pointed out earlier code-of-conduct-guide.md should be replaced with guide-code-of-conduct.md.

Reply:
I think I've standardized them all now, so we should be good.